frontend/src/Frontend/Wallet.hs

```diff
@@ -81,10 +86,18 @@ import Frontend.KadenaAddress
 import Frontend.Storage
 import Frontend.Network
 
+import Frontend.UI.Widgets
+
+accountIsCreated :: Account key -> AccountCreated
+accountIsCreated = maybe AccountCreated_No (const AccountCreated_Yes) . _account_balance
+
+accountCreatedYesNo :: Account key -> a -> a -> a
+accountCreatedYesNo a yes no = case accountIsCreated a of
```

Comment:
How about `accountCreatedBool :: a -> a -> Account key -> a` (and flip the yes/no order) to match `bool` ?